spec/views/user_show_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe 'User Features', type: :feature do
+  before do
+    @doctor = User.create!(id: 5, name: 'Mauricio',
+                           photo: 'https://upload.wikimedia.org/wikipedia/commons/4/41/Profile-720.png',
+                           bio: 'Mastology Doctor')
+    @advent_hospital_patient = User.create!(id: 6, name: 'Lily',
+                                            photo:
+    'https://upload.wikimedia.org/wikipedia/commons/4/41/Profile-720.png',
+                                            bio: 'Advent Hospital patient')
+
+    Post.create!(title: 'Introductory Post', text: 'Hello, everyone!', comments_counter: 0,
+                 likes_counter: 2, author: @doctor)
+    Post.create!(title: 'Another intro', text: 'Greetings!', comments_counter: 0,
+                 likes_counter: 2, author: @doctor)
+    Post.create!(title: 'Getting Started', text: 'Hi there!', comments_counter: 0,
+                 likes_counter: 2, author: @doctor)
+    Post.create!(title: 'Intro to the Community', text: 'Hi, all!', comments_counter: 0,
+                 likes_counter: 2, author: @doctor)
+
+    @patient_post = Post.create!(title: 'Patient Introduction', text: 'Hey, friend!',
+                                 comments_counter: 0, likes_counter: 2, author: @advent_hospital_patient)
+  end
+
+  describe 'User Profile Page' do
+    it 'displays specific user names' do
+      visit user_path(@doctor)
+      expect(page).to have_content('Mauricio')
+      visit user_path(@advent_hospital_patient)
+      visit(page).to have_content('Lily')
+    end
+
+    it 'displays users\'profiles' do
+      visit user_path(@doctor)
+      expect(page).to have_css("img[src*='#{@doctor.photo}']")
+      visit user_path(@advent_hospital_patient)
+      expect(page).to have_csss("img[src*='#{@advent_hospital_patient.photo}']")
+    end
+
+    it 'displays the recent 3 posts for each user' do
+      visit user_path(@doctor)
+
+      expect(@doctor.three_most_recent_posts.count).to have_content(3)
+      visit user_path(@advent_hospital_patient)
+      expect(@advent_hospital_patient.posts.count).to have_content(1)
+    end
+
+    it 'displays the number of posts for each user' do
+      visit user_path(@doctor)
+      expect(@doctor.posts.count).to have_content(4)
+      visit user_path(@advent_hospital_patient)
+        .expect(@advent_hospital_patient.posts.count).to have_content(1)
+    end
+
+    it 'displays the user\'s bio' do
+      visit user_path(@doctor)
+      expect(@doctor.bio).to have_content('Mastology Doctor')
```

Comment:
- [ ] A number of the view specs are failing for user show and post index views, let's ensure all test are passing!

**Failing tests:**
![image](https://github.com/IvonneBenitesRodriguez/rails_blog/assets/12745474/9c10c741-d9c3-47d0-a333-91cd361f1563)

Reply:
Dear @rloterh 👋🏾,
Very nice to meet you,
thank you for your suggestions,
I appreciate it, I already modified it.
Best wishes,